experiment_buddy/experiment_buddy.py

```diff
@@ -195,7 +270,14 @@ def watch(self, *args, **kwargs):
         self.run.watch(*args, **kwargs)
 
     def close(self):
-        pass
+        self.dump()
+        if hasattr(self, "orion_experiment"):
+            value = self.run.summary.get(self.sweep_objective)
+            if value is None:
+                warnings.warn(f"Wandb did not compute summary for {self.sweep_objective}.")
+                value = 0
+            result = [{"name": self.sweep_objective, "type": "objective", "value": value}]
+            report_to_orion(self.orion_experiment, trial_id=self.run.config["^trial_id"], result=result)
```

Comment:
do we know trial id? can you skip self.run.config?

Reply:
is in "global" hyperparameters , and in the json file `task=$task_id.json` that is the local dump of the parameter. which one is more appropriate?